resources/js/components/app-grid.js

```diff
@@ -207,7 +207,11 @@ class AppGrid extends LitElement {
             window.location.href = url
         }
     }
-
+    addOrUpdateQueryParam(url, key, value) {
+    let urlObj = new URL(url)
```

Comment:
@Muniraj132, please add indentation to this method. In your PHPStorm settings, turn on Prettier to allow for linting.

Reply:
@incraigulous, I have ensured the correct indentations are in place according to our code style guidelines. Additionally, I’ve configured Prettier in PHPStorm for consistent formatting. Thank you